### PR TITLE
Various spelling and documentation fixes

### PR DESCRIPTION
**Pull request by alexwlchan**
- Fix capitalisation of the following names: AppArmor, BitBucket,
  Fedora, GitHub, Package Control, SoftLayer, VirtualBox
- Fix spacing of the following phrases: OS X, Red Hat
- Remove stray spaces
- Replace goo.gl short link with full link
- Various spelling corrections

Signed-off-by: Alex Chan alex@alexwlchan.net

### CODE REVIEW

File: docs/man/docker-create.1.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ IMAGE [COMMAND] [ARG...]
 **--cgroup-parent**=""
    Path to cgroups under which the cgroup for the container will be created. If the path is not absolute, the path is considered to be relative to the cgroups path of the init process. Cgroups will be created if they do not already exist.
 
-**--cpu-peroid**=0
+**--cpu-period**=0
```
Comment (alexwlchan) [inline docs/man/docker-create.1.md]: This is just a typo in the man page, not a misspelt command-line flag.

The flag is defined in [/api/client/build.go](https://github.com/docker/docker/blob/master/api/client/build.go) on line 57, and is spelt “period” not “peroid”.

**Comment by moxiegirl** [comment]
Nicely done. Thanks for the contribution. 

LGTM

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

**Comment by calavera** [comment]
This needs to be rebased, sorry.

**Comment by thaJeztah** [comment]
Very LGTM (but needs a rebase)

**Comment by SvenDowideit** [comment]
LGTM

**Comment by thaJeztah** [comment]
@alexwlchan can you rebase your PR, then this is ready to be merged. Please leave a comment after you rebased, to be sure we get notified :)

**Comment by thaJeztah** [comment]
I just realize I'm typing on my phone; you may have rebased already, but GitHub doesn't show properly on mobile; I'll check when I'm at my computer :smile:

**Comment by icecrime** [comment]
@thaJeztah Do you mind carrying please?

**Comment by thaJeztah** [comment]
@icecrime yup, will do. Will be after the weekend, as I'm away right now :)

**Comment by moxiegirl** [comment]
@thaJeztah I'm going to have my big merge for tooling coming in on Monday. Just a heads up...I'll make sure to wait for your fixes on this.

**Comment by thaJeztah** [comment]
@moxiegirl if you have time to incorporate these fixes, that's also find with me, I'm away for the weekend, so don't have access to my computer until somewhere during Monday. Just let me know :smile:

**Comment by moxiegirl** [comment]
@thaJeztah Yeah, I think I should probably do it ---  I have to rebase before my own PRs anyway.  Enjoy your weekend Seb, no worries. I'll carry it.

**Comment by thaJeztah** [comment]
Thanks @moxiegirl, happy to help out another time!

**Comment by alexwlchan** [comment]
Gah, I let this languish, didn’t I? Sorry, I’ve been on holiday and not had much computer access for a while. I had a quick go before I left, but mucked up the rebase.

I’ll try to get this into a mergeable-state within the next day or so.

**Comment by moxiegirl** [comment]
carried by https://github.com/docker/docker/pull/13927

**Comment by thaJeztah** [comment]
no worries, @alexwlchan! thanks for creating this PR, @moxiegirl will carry it as she had to make additional changes in these parts of the documentation.

:+1:

**Pull request closed without merging**